openlibrary/plugins/openlibrary/js/availability.js

```diff
@@ -165,14 +165,14 @@ function initAvailability() {
                             $(`${selector}[data-key=${book_key}]`)
                                 .attr('href', $(`${selector}[data-key=${book_key}]`).attr('data-key'))
                             $(`${selector}[data-key=${book_key}]`)
-                                .attr('title', 'Check Availability');
+                                .attr('title', 'Check');
```

Comment:
Let's not change this, as it will provide patron's more information when they are hovering over the button.

Reply:
Okay doing both the changes